skia.lua

```diff
@@ -451,7 +444,6 @@ project "skia"
     configuration { "windows" }
 
       defines {
-        "SK_CPU_SSE_LEVEL=31",
```

Comment:
where did these end up moving to?

Reply:
@dkalowsk skia implicitly finds them now in the preconfig file. I left a note for this https://github.com/Esri/skia/pull/5/files#diff-3f8f333bcb2cbb7d4f032e3366973615R136. Glad they are able to figure it out now so all we need to do is pass the build files.